src/client/pages/CheckoutDetails/CheckoutDetails.tsx

```diff
@@ -1,6 +1,23 @@
 import React from 'react'
+import { useParams } from 'react-router-dom'
+import { useTextKeys } from 'utils/textKeys'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
 import { CheckoutPageWrapper } from 'components/checkout/CheckoutPageWrapper'
+import { Footer } from 'components/checkout/Footer'
 
 export const CheckoutDetails = () => {
-  return <CheckoutPageWrapper>{}</CheckoutPageWrapper>
+  const textKeys = useTextKeys()
+  const { path: localePath } = useCurrentLocale()
+  const { id: quoteCartId } = useParams<{ id: string }>()
```

Comment:
Didn't we create a custom hook for getting quote cart ID from URL?

Reply:
I haven't seen anything like that, I think maybe we just talked about creating that? Or I missed it 🙃

Reply:
Hmm nope, you are right. When we do this a third time we can extract it into a hook then haha